person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ] Kindly move these lines before the initialize method, you can see this article of [how to use...](https://www.rubyguides.com/2018/11/attr_accessor/)